### DISCUSSION TITLE
Need Suggestions

### DISCUSSION THREAD

**Discussion by cbpanteon** [mebjas/html5-qrcode#247] [Q&A]
1) I have used html5-qrcode in one of my project. I observe that there is a slit delay to open up the video frame. Can we remove/reduce that delay? 
2) One more thing I have observed: Before loading the video frame, it will load a video icon and then load the video frame. can we remove that video icon? please find the attachment for video icon.

![Screenshot from 2021-07-02 11-20-47](https://user-images.githubusercontent.com/15908222/124227429-46ec5400-db28-11eb-9f19-74ccf370aca6.png)

**Comment by ROBERT-MCDOWELL** [comment]
this is a browser limitation for sure. Please provide your browser specs.

**Reply by cbpanteon** [reply]
Thanks, Robert for the quick response.
We have our application. We are using Android web view to load the site.

**Comment by ROBERT-MCDOWELL** [comment]
your app should have something in the manifest that allows autoplay

**Reply by cbpanteon** [reply]
No. Video frame is coming. But it will take some delay to load. Is there any way that I can reduce that delay?
Ane before video frame load it will load the previously attached icon and after that, it will load video frame. Is there any way I can remove that icon and open video frame directly?
